File: src/main/java/org/profit/util/IntegerUtils.java
package org.profit.util;

/**
 * @author TangYing
 */
public class IntegerUtils {

    /**
     * Generate min ~ max random number
     *
     * @param min
     * @param max
     * @return
     */
    public static int generateRandomSeed(int min, int max) {
        if (min == max) {
            return min;
        }

        double seed = Math.random() * (max - min) + min;
        return (int) Math.round(seed);
    }
}

File: src/main/java/org/profit/util/IntegerUtils.kt
package org.profit.util

/**
 * @author TangYing
 */
object IntegerUtils {
    /**
     * Generate min ~ max random number
     *
     * @param min
     * @param max
     * @return
     */
    fun generateRandomSeed(min: Int, max: Int): Int {
        if (min == max) {
            return min
        }
        val seed = Math.random() * (max - min) + min
        return Math.round(seed).toInt()
    }
}